lib/constants.js

```diff
@@ -65,7 +65,6 @@ module.exports.DOCUMENT_TYPE = {
   RESIDENCE_PERMIT: '5',
   KBIS: '7',
   DRIVER_LICENCE: '11',
-  SOCIAL_SECURITY: '14',
   FAMILY_BOOKLET: '15',
   INSURANCE: '16'
```

Comment:
Are we sure `FAMILY_BOOKLET` and `INSURANCE` are still mapped to 15 and 16 ?

Reply:
https://documentation.lemonway.com/reference/accounts_documentsuploadpost
Documentation states types `15` and `16` are free slots for "other document types"